server/base/src/main/java/org/apache/accumulo/server/util/MasterMetadataUtil.java

```diff
@@ -253,4 +253,27 @@ public static void updateTabletDataFile(ServerContext context, KeyExtent extent,
 
     tablet.mutate();
   }
+
+  public static void updateLastLocation(ServerContext context, KeyExtent extent, FileRef path, DataFileValue dfv, MetadataTime time,
+                                        String address, ZooLock zooLock, TServerInstance lastLocation) {
+
+    TabletMutator tablet = context.getAmple().mutateTablet(extent);
+    if(dfv.getNumEntries() > 0) {
```

Comment:
Why are we passing this in and checking its number of entries?

Reply:
That is what previous functions used and was just copied over. I wasn't sure about the intentions but I agree that it is not needed.